elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -204,6 +216,18 @@ default String getHibernate5ConfigPath() {
      * @param servletContextHandler ServletContextHandler in use by Elide standalone.
      */
     default void updateServletContextHandler(ServletContextHandler servletContextHandler) {
-        // Do nothing
+        ServletHolder jerseyServlet = servletContextHandler.addServlet(ServletContainer.class, "/swagger");
```

Comment:
We should do this in the core code rather than the extension point.  Look at how we optionally configure JSON-API and GraphQL.  Swagger should be similar.